content/ko/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -37,8 +37,8 @@ OwnerReference가 없거나, OwnerReference가 {{< glossary_tooltip term_id="con
 레플리카셋은 지정된 수의 파드 레플리카가 항상 실행되도록 보장한다.
 그러나 디플로이먼트는 레플리카셋을 관리하고 다른 유용한 기능과 함께
 파드에 대한 선언적 업데이트를 제공하는 상위 개념이다.
-따라서 우리는 사용자 지정 오케스트레이션이 필요하거나 업데이트가 전혀 필요하지 않은 경우라면
-레플리카셋을 직접적으로 사용하기 보다는 디플로이먼트를 사용하는 것을 권장한다.
+따라서 별도의 사용자 정의(custom) 오케스트레이션이 필요하거나 업데이트가 전혀 필요 없는 경우가 아니라면,
+레플리카셋을 직접 사용하기 보다는 디플로이먼트를 사용하는 것을 권장한다.
```

Comment:
@bcking92 안녕하세요~~ 기여에 감사드립니다.

[원문]
> Therefore, we recommend using Deployments instead of directly using ReplicaSets, unless you require custom update orchestration or don't require updates at all.

마지막 문장에 따르면 다음과 같은 것으로 보입니다.
- 아래의 경우를 제외하면, 레플리카셋을 직접 사용하기보다는 디플로이먼트를 사용하는 것을 권장
  1. custom update orchestration이 필요한 경우, 또는
  2. 업데이트가 전혀 필요 없는 경우

중요한 오류를 잘 잡아 주셨네요~~

약간의 수정 제안을 남겨 봅니다.
확인해 보시고, 선택적으로 반영하시면 되겠습니다. 😊

```suggestion
따라서 별도의 사용자 정의(custom) 업데이트 오케스트레이션이 필요한 경우 또는 업데이트가 전혀 필요 없는 경우가 아니라면,
레플리카셋을 직접 사용하기보다는 디플로이먼트를 사용하는 것을 권장한다.
```